src/entities/user.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, Column, PrimaryGeneratedColumn, Unique } from 'typeorm';
+
+@Entity('users')
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  password: string;
```

Comment:
Maybe we should add `Exclude()` and use `ClassSerializerInterceptor` globally to make sure the password never serialized.